### ISSUE TITLE
Remove search-replace extension

### ISSUE THREAD

**Issue by janschulz** [ipython-contrib/IPython-notebook-extensions#538]
This is now in the notebook.

https://github.com/ipython-contrib/IPython-notebook-extensions/tree/master/nbextensions/usability/search-replace

**Comment by juhasch** [comment]
I agree.

**Comment by jcb91** [comment]
I see what you mean, but something I've found missing from the notebook's search-replace is the ability to go to a particular find result - it only seems to function as a global-replace. Though perhaps I should just write that into notebook core :confused:

**Comment by janschulz** [comment]
@jcb91 Please do :-) Until then we can keep the extensions (and the PR open).

**Comment by juhasch** [comment]
I think the search-replace functionality should be in the notebook. Having two different implementations will confuse people.

**Comment by jcb91** [comment]
@juhasch fair enough, I'm happy for it to be removed, I can always keep a local copy for my own use in the meantime :smiley:

**Comment by juhasch** [comment]
There is no hurry to delete the extension. Let's keep the PR open and wait for the notebook to fix it.
